functional-tests/views/index.pug

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
```

Comment:
Why is this a pug file?

Reply:
since we're setting up the preview expiring embed using an express app, it uses pug to pass in the variables rather than a straight html file